lib/OptimizationBBO/src/OptimizationBBO.jl

```diff
@@ -9,6 +9,11 @@ abstract type BBO end
 SciMLBase.requiresbounds(::BBO) = true
 SciMLBase.allowsbounds(::BBO) = true
 SciMLBase.supports_opt_cache_interface(opt::BBO) = true
+SciMLBase.requiresgradient(::BBO) = false
```

Comment:
Since the default value is false, you can only declare the ones you need to set to true here, so for example you'd not need to define anything here in OptimzationBBO since its all false